packages/rest/src/parser.ts

```diff
@@ -60,7 +61,7 @@ async function buildOperationArguments(
   pathParams: PathParameterValues,
   body: RequestBody,
   globalSchemas: SchemasObject,
-  options: RequestBodyValidationOptions = {},
+  options: RequestBodyValidationOptions,
```

Comment:
can we keep `={}` if not using `AjvFactoryProvider.defaultValidationOptions`?

Reply:
It is not obvious here, but the option is of a non-exported function which is called in only one place with the default as `AjvFactoryProvider.defaultValidationOptions`, so I removed it to reduce redundancy.

Making the default `{}` will undo the changes of this PR, IF this function happens to be called without an `option` in another place in future, so let's keep it as it is. Remember, default `{}` is what caused the configured custom error messages to not be used 😄

Reply:
Btw, I have made `DEFAULT_AJV_VALIDATION_OPTIONS` the default now.